src/Features/Activity/ActivityHeader/ActivityHeader.tsx

```diff
@@ -29,12 +33,16 @@ function ActivityHeader({
   const emoji = successRatePercentage > 79 ? "🙌" : successRatePercentage > 49 ? "😮" : "😨";
 
   return (
-    <FeatureHeader includeBorder={false}>
-      <div className={styles.container}>
-        <hgroup>
-          <p className={styles.subtitle}>This is all of the</p>
-          <h1 className={styles.title}>Activity</h1>
-        </hgroup>
+    <Header
+      className={styles.container}
+      includeBorder={false}
+      header={
+        <>
```

Comment:
@LucasGrimauth I think we would want to keep the <hgroup> tag here

Reply:
@BenjaminRuby the header prop in the FeatureHeader is already involved in a `hgroup` tag.

Reply:
@LucasGrimauth ah I missed that. I think I assumed that the `hgroup` would be passed in by the consumer since there are instances where we use only one header, and instances where we use multiple. It seems slightly overkill to use an hgroup with one header, although it's probably not a big deal.

@timrbula any thoughts on the above?

Reply:
Reading the usage notes again on MDN, it says it has been removed from the HTML5 specification (so a good while ago) and only has theoretically use even if it's supported by browsers. I am also seeing that its suggested to use h1 w/ a p tag instead of h1, h2 grouped together. that would actually work better for things like flow where the "h2" comes first, without having to resort to a wrapper element to keep the html structure the same with h1 then h2, but the appearance be h2 then h1.

So I can make these updates then we can look at where we are using the hgroup tag. I think I was the one that started using them a lot and didn't see the usage guidelines so my bad on that yall.